service/src/main/java/uk/gov/hmcts/reform/fpl/model/Respondent.java

```diff
@@ -44,6 +44,7 @@
     private String legalRepresentation;
 
     private RespondentSolicitor solicitor;
+    private int policyReference;
```

Comment:
that can be removed, we don't need the reference, since we are not deleting elements after submission